xarray/util/generate_aggregations.py

```diff
@@ -209,7 +277,8 @@ def {method}(
     example="""\n
         Specify ``min_count`` for finer control over when NaNs are ignored.
 
-        >>> {calculation}(skipna=True, min_count=2)""",
+        >>> {calculation}(skipna=True, min_count=2)
```

Comment:
In general, these strings were tweaked to make sure pre-commit was happy so its unlikely these changes are necessary.